docs/general/dql/index.rst

```diff
@@ -1,12 +1,13 @@
 .. highlight:: psql
-.. _sql_dql:
+
+.. _dql:
 
 ========
 Querying
 ========
 
-This section provides an overview on how to query documents using SQL.  See
-:ref:`sql_ddl` for information about `Table creation` and other Data Definition
+This section provides an overview on how to query documents using SQL. See
+:ref:`ddl` for information about `Table creation` and other Data Definition
```

Comment:
Did you intend to link `Table creation` anywhere?

Reply:
that was there before I made this change. however, I took the opportunity to improve the index pages for DDL, DML, and DQL. thanks!